resources/configurationFiles/analysisSNVCalling.xml

```diff
@@ -182,10 +187,10 @@
         <tool name="snvFilter" value="filter_vcf.sh" basepath="snvPipeline">
             <resourcesets>
                 <rset size="t" memory="1" cores="1" nodes="1" walltime="1" queue="devel"/>
-                <rset size="s" memory="1" cores="1" nodes="1" walltime="1"/>
-                <rset size="m" memory="1" cores="1" nodes="1" walltime="2"/>
-                <rset size="l" memory="1" cores="1" nodes="1" walltime="3"/>
-                <rset size="xl" memory="4" cores="1" nodes="1" walltime="4"/>
+                <rset size="s" memory="3" cores="3" nodes="1" walltime="7"/>
+                <rset size="m" memory="3" cores="3" nodes="1" walltime="9"/>
+                <rset size="l" memory="3" cores="3" nodes="1" walltime="11"/>
+                <rset size="xl" memory="4" cores="3" nodes="1" walltime="13"/>
             </resourcesets>
```

Comment:
That's quite an increase -- up to 9 hours longer runtime.

Reply:
Oh no. For OTP the runtime really does not matter. But for the users who are waiting for the data.

Your comment about the VEP column and the column order I don't understand.